cmd/agent/app/integrations_windows.go

```diff
@@ -30,6 +31,8 @@ func init() {
 	pd, err := winutil.GetProgramDataDir()
 	if err == nil {
 		tufPipCachePath = filepath.Join(pd, "Datadog", "repositories", "cache")
+	} else {
+		winutil.LogEventViewer(config.ServiceName, 0x8000000F, tufPipCachePath)
```

Comment:
Not a huge fan of these magic numbers, but I can see we use them elsewhere. At some point we should maybe use consts and bit masks for the log level and message number. Can we create a card for that?